comments-by-guideline-and-success-criterion.md

```diff
@@ -1063,15 +1063,21 @@ WCAG 2.2 has made this success criterion obsolete and removed it as a requiremen
 
 WCAG 2.0 and 2.1 are incorporated, either directly or by reference, into other standards. Therefore, the application of 4.1.1 Parsing to non-web documents and software is to follow the new guidance provided in the [WCAG 2.0 Editorial Errata](https://www.w3.org/WAI/WCAG20/errata/#editorial) and the [WCAG 2.1 Editorial Errata](https://www.w3.org/WAI/WCAG21/errata/#editorial) which states the following:
 
->This Success Criterion should be considered as always satisfied for any content using HTML or XML.
+>Success Criterion 4.1.1 was originally adopted to address problems that assistive technology had directly parsing HTML. Since this criterion was written, the HTML Standard has adopted specific requirements governing how user agents must handle incomplete tags, incorrect element nesting, duplicate attributes, and non-unique IDs.
+>
+>Although the HTML Standard treats some of these cases as non-conforming for authors, it is considered to "allow these features" for the purposes of this Success Criterion because the specification requires that user agents support handling these cases consistently. In practice, this criterion no longer provides any benefit to people with disabilities in itself.
+>
+>Issues such as missing roles due to inappropriately nested elements or incorrect states or names due to a duplicate ID are covered by different Success Criteria and should be reported under those criteria rather than as issues with 4.1.1.
+>
+>This criterion should be considered as always satisfied for any content using HTML or XML.
```

Comment:
The crux of the problem is here. Yes, for HTML and XML this criterion is always satisfied. But not every markup language is HTML / XML based. Far from it. I think the 4.1.1 content from the previous WCAG2ICT is still relevant here.

Reply:
@maryjom did you want to add in @WilcoFiers' edits or would you like me to approve his request?

Reply:
@ChrisLoiselle This is an unresolved discussion. I am meeting with Wilco and Dan tomorrow morning on this to decide what to do. The 2013 WCAG2ICT content that is suggested to be added has a lot of outdated content in it. I still have concerns with requiring this SC.